UI/window-youtube-actions.cpp

```diff
@@ -235,6 +235,19 @@ OBSYoutubeActions::OBSYoutubeActions(QWidget *parent, Auth *auth)
 	this->resize(this->width() + 200, this->height() + 120);
 #endif
 	valid = true;
+
+	OBSBasic *main = OBSBasic::Get();
+	bool settingsSaved =
+		config_get_bool(main->basicConfig, "YouTube", "SettingsSaved");
+
+	if (settingsSaved) {
+		QMessageBox::StandardButton button = OBSMessageBox::question(
+			main, QTStr("YouTube.Actions.ReuseSettings.Title"),
+			QTStr("YouTube.Actions.ReuseSettings.Description"));
```

Comment:
I think it'd be worth giving the user some context as to what their previous settings were in this dialog so that they don't click "Yes" and then regret their decision.

Reply:
Any ideas on what the text should be?

Reply:
This auto fills the inputs. The users can still change them if they want.